### PR TITLE
Add application scaffolding 

### PR DESCRIPTION
**Pull request by jemoreira**

### CODE REVIEW

File: controller.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
```
Comment (ser-io) [inline controller.go:7]: nit: fix indentation.

Comment (jemoreira) [reply controller.go:7]: Done

File: controller.go [hunk 2]
```diff
@@ -15,26 +15,120 @@
...
+	"strconv"
+
+	"github.com/gorilla/mux"
 )
 
 // The controller implements the web API of the cloud orchestrator. It parses
 // and validates requests from the client and passes the information to the
 // relevant modules
 type Controller struct {
+	ss SignalingServer
```
Comment (ser-io) [inline controller.go:31]: nit: should we have more descriptive field names? Go advocates for one letter names when scope is smaller, but field names could be used anywhere.

File: controller.go [hunk 3]
```diff
@@ -126,6 +117,19 @@ func indexHandler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintln(w, "Home page")
 }
 
+// Get an int form value from the request
```
Comment (ser-io) [inline controller.go:120]: nit: let's make clear that `default` is only used if the field name does not exist, if it exists and it's an invalid value an error will be returned.

File: controller.go (hunk 2 above)
Comment (jemoreira) [reply controller.go:31]: absolutely!

File: controller.go (hunk 3 above)
Comment (jemoreira) [reply controller.go]: Done

**Review by ser-io — APPROVED** [review]

**Pull request merged**